client/components/account-balances/test/index.test.tsx

```diff
@@ -138,19 +138,7 @@ const createMockOverview = (
 			fee_percentage: 0,
 			status: 'paid',
 		},
-		nextScheduled: {
-			id: '456',
-			type: 'deposit',
-			amount: 0,
-			automatic: true,
-			currency: null,
-			bankAccount: null,
-			created: Date.now(),
-			date: Date.now(),
-			fee: 0,
-			fee_percentage: 0,
-			status: 'estimated',
-		},
+		nextScheduled: undefined,
```

Comment:
The `nextScheduled` wasn't used for this test suite, made apparent by the lack of additional changes required in this or related test files.

Reply:
Assuming we can't delete the `nextScheduled` key from this test data for some reason?